lib/parsers/mp3_parser.rb

```diff
@@ -68,9 +69,10 @@ def call(raw_io)
     # will terminate here. Same with PNGs. In the future
     # we should implement "confidence" for MP3 as of all our formats
     # it is by far the most lax.
-    header = safe_read(io, 8)
+    header = safe_read(io, 12)
     return if header.start_with?(ZIP_LOCAL_ENTRY_SIGNATURE)
     return if header.start_with?(PNG_HEADER_BYTES)
+    return if header.start_with?(WEBP_HEADER_REGEX)
```

Comment:
Looks great! My only concern is that regexes are only [introduced](https://ruby-doc.org/core-2.5.1/String.html#method-i-start_with-3F) with this method in Ruby 2.6. I see we still support Ruby 2.5. Not sure how the tests passed, but could you double check this locally, with Ruby 2.5, with a file that satisfies this condition?

Reply:
It looks to me from that link you sent as if there was a case for regexes pre-2.6:
```
case T_REGEXP:
    {
        bool r = rb_reg_start_with_p(tmp, str);
        if (r) return Qtrue;
    }
```

Installing 2.5 now to be sure though!

Reply:
Confirmed that it works with 2.5.

Reply:
Appreciate you checking it 🙏